app.rb

```diff
@@ -0,0 +1,189 @@
+# Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following:
+#   List all books.
+#   List all people.
+#   Create a person (teacher or student, not a plain Person).
+#   Create a book.
+#   Create a rental.
+#   List all rentals for a given person id.
+#   In your main.rb define the entry point, this will be a method called main
+# that is invoked at the end of your file. This method should do the following:
+#   Present the user with a list of options to perform.
+#   Lets users choose an option.
+#   If needed, ask for parameters for the option.
+#   Have a way to quit the app.
+
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_people
+    if @people.length.zero?
+      puts 'There are no people in the library'
+    else
+      @people.each do |person|
+        if person.instance_of?(Student)
+          puts "[Student]: Name: #{person.name}, Id: #{person.id}, Age: #{person.age}"
+        else
+          puts "[Teacher]: Name: #{person.name}, Id: #{person.id}, Age: #{person.age}"
+        end
+      end
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
```

Comment:
- [ ] Kindly make sure to fix the error as shown in the below video. When I click an option for example 2 it displays the entire list and when I click 2 again it displays the required function with it. Make sure that app works smoothly.

https://user-images.githubusercontent.com/70269507/209128275-82c171fe-060d-4763-8d19-b7a4031ba778.mov